index.html

```diff
@@ -519,6 +519,79 @@ <h3>LinkedDataSignature2016</h3>
           </pre>
       </section>
 
+
+      <section id="MerkleProof2019" about="https://w3id.org/security#MerkleProof2019">
+
+        <h3>MerkleProof2019</h3>
+          <p>
+A Linked Data signature is used for digital signatures on RDF Datasets.
+The default canonicalization mechanism is specified in the RDF Dataset
+Normalization specification, which effectively deterministically names all
+unnamed nodes. The default signature mechanism
+uses a SHA-256 digest and ECDSA to perform the digital signature. More information at
+              <a href="https://w3c-ccg.github.io/lds-merkle-proof-2019/">https://w3c-ccg.github.io/lds-merkle-proof-2019/</a>
+          </p>
+          <dl>
+            <dt>Status</dt>
+            <dd property="vs:term_status">stable</dd>
+            <dt>Parent Class</dt>
+            <dd>LinkedDataSignature</dd>
+            <dt>Expected properties</dt>
+            <dd>proofValue</dd>
+            <dt>Signature Properties</dt>
+            <dd>
+              <dl>
+                <dt>Default Canonicalization Algorithm</dt>
+                <dd>
+<a rel="canonicalizationAlgorithm"
+  href="https://w3id.org/security#GCA2015">https://w3id.org/security#GCA2015</a>
+                </dd>
+                <dt>Default Signature Algorithm</dt>
+                <dd>
+<a rel="signatureAlgorithm"
+  href="http://www.w3.org/2000/09/xmldsig#rsa-sha256">
+http://www.w3.org/2000/09/xmldsig#rsa-sha256</a>
```

Comment:
The signature algorithm is different here... you need to point to the signature algorithm in the MerkleProof2019 spec.

Reply:
see point above? I think it answers the same thing.

Reply:
unless you are signing with ECDSA or RSA as part of contructing a MerkleProof2019, it does not.

Reply:
@dsemenovsky -- the thing above states that you are using RSA to do the digital signature. I expect that you're not actually doing that. Correct?

Reply:
No, I am not using RSA anywhere.
What should serve as a signature algorithm here?

Should we define that as ECDSA because that how the blockchain transaction gets signed?

Reply:
Yes, it should be defined as ECDSA... is there a URL that identifies ECDSA? I don't know for sure, but I'd look at the IANA registries first... JOSE Web Algorithms might give you a URL... don't know if there is a direct HTTP link to the exact ECDSA algorithm.

Reply:
Found it! https://tools.ietf.org/html/rfc6931#section-2.3.6

Looks like the URL you're looking for is:

http://www.w3.org/2001/04/xmldsig-more#ecdsa-sha256